### PR TITLE
Update /submission/config call 

### PR DESCRIPTION
**Pull request by sdevalapurkar-bcgov**
# Description

This PR includes the following proposed change(s):

- Update `/submission/config` call to return `clientAppName` instead of `clientApplication` object
- Also update to return `csoBaseUrl` instead of that env var being on frontend

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation

## How Has This Been Tested?

- local
- unit tests
- postman

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/efiling-api/jag-efiling-api.yaml [hunk 1]
```diff
@@ -480,8 +480,12 @@ components:
       properties:
         navigation:
           $ref: "#/components/schemas/Navigation"
-        clientApplication:
-          $ref: "#/components/schemas/ClientApplication"
```
Comment (sdevalapurkar-bcgov) [inline src/backend/efiling-api/jag-efiling-api.yaml:484]: dont we need to remove the `ClientApplication` schema??

File: src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java [hunk 2]
```diff
@@ -290,10 +290,12 @@ public SubmissionApiDelegateImpl(
 
         GetSubmissionConfigResponse response = new GetSubmissionConfigResponse();
 
-        response.setClientApplication(fromCacheSubmission.get().getClientApplication());
+        response.setClientAppName(fromCacheSubmission.get().getClientApplication().getDisplayName());
```
Comment (sdevalapurkar-bcgov) [inline src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java:293]: question: do we still need the clientApplication object with a display name property or can we just have a value for clientAppName?

File: src/backend/efiling-api/src/main/resources/application.yml [hunk 3]
```diff
@@ -23,6 +23,7 @@ jag:
       demo: ${DEMO_MODE:false}
     navigation:
       baseUrl: ${NAVIGATION_BASE_URL:http://localhost:3000/efiling}
+      csoBaseUrl: ${CSO_BASE_URL:http://localhost/cso}
```
Comment (sdevalapurkar-bcgov) [inline src/backend/efiling-api/src/main/resources/application.yml:26]: shouldnt this be:
https://dev.justice.gov.bc.ca/cso

File: src/backend/efiling-api/jag-efiling-api.yaml (hunk 1 above)
Comment (alexjoybc) [reply src/backend/efiling-api/jag-efiling-api.yaml:484]: no we have it on the generate url

File: src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java (hunk 2 above)
Comment (alexjoybc) [reply src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java:293]: yes we do lol

File: src/backend/efiling-api/src/main/resources/application.yml (hunk 3 above)
Comment (alexjoybc) [reply src/backend/efiling-api/src/main/resources/application.yml:26]: not the greatest, will have to think about it lol

**Comment by sdevalapurkar-bcgov** [comment]
LGTM 🚀

**Pull request merged**